### PR TITLE
Finished log in and log out functionality

### PR DESCRIPTION
**Pull request by olliecase-green**
Styled pages, linked them and added log in cookies and functionality

### CODE REVIEW

File: knowledge-challenge-app/src/Components/Networking.js [hunk 1]
```diff
@@ -18,19 +17,19 @@ export default class Network {
...
-  //       credentials: "include",
-  //       headers: { "Content-Type": "application/json" },
-  //       body: JSON.stringify({
-  //         username: username,
-  //         password: password,
-  //       }),
-  //     });
-  //     let json = await response.json();
-  //     return json;
-  //   }
```
Comment (zaynshah) [inline knowledge-challenge-app/src/Components/Networking.js:33]: Is this needed?

File: knowledge-challenge-app/src/App.js [hunk 2]
```diff
@@ -1,21 +1,57 @@
...
+        <Route path="/homepage">
+          <Homepage setIsLoggedIn={setIsLoggedIn} />
         </Route>
+
+        <Route
+          path="/dashboard"
+          element={
+            isLoggedIn ? <StudentDashboard /> : <Redirect to="/homepage" />
+          }
+        ></Route> */}
```
Comment (zaynshah) [inline knowledge-challenge-app/src/App.js:54]: Is this needed?

**Review by FahmidulHaquee — COMMENTED** [review]
Good changes on the header, dashboard and log-in modal.

**Review by zaynshah — APPROVED** [review]
looks good!

**Pull request merged**